### ISSUE TITLE
pytest doesn't run all the tests

### ISSUE THREAD

**Issue by kragniz** [boatd/boatd#19]
It looks like pytest isn't discovering all the tests:
```
collected 30 items 

boatd/tests/test_boat.py::TestBoat::test_get_heading PASSED
boatd/tests/test_boat.py::TestBoat::test_get_pony PASSED
boatd/tests/test_boat.py::TestBoat::test_active PASSED
boatd/tests/test_config.py::TestConfig::test_load_yaml PASSED
boatd/tests/test_config.py::TestConfig::test_load_json PASSED
boatd/tests/test_config.py::TestConfig::test_port PASSED
boatd/tests/test_config.py::TestConfig::test_driver PASSED
boatd/tests/test_config.py::TestConfig::test_set_attr PASSED
boatd/tests/test_config.py::TestConfig::test_get PASSED
boatd/tests/test_config.py::TestConfig::test_get_invalid PASSED
boatd/tests/test_driver.py::TestDriver::test_driver_file PASSED
boatd/tests/test_driver.py::TestDriver::test_loading_driver PASSED
boatd/tests/test_driver.py::TestDriver::test_pony_exists PASSED
boatd/tests/test_driver.py::TestDriver::test_pony_runs PASSED
boatd/tests/test_driver.py::TestDriver::test_heading PASSED
boatd/tests/test_driver.py::TestDriver::test_wind_speed PASSED
boatd/tests/test_driver.py::TestDriver::test_handler_decorators PASSED
boatd/tests/test_driver.py::TestDriver::test_empty_hardware PASSED
boatd/tests/test_driver.py::TestDriver::test_set_rudder PASSED
boatd/tests/test_driver.py::TestDriver::test_set_rudder_multiple PASSED
boatd/tests/test_driver.py::TestDriver::test_bad_driver PASSED
boatd/tests/test_nmea.py::test_hdm PASSED
boatd/tests/test_nmea.py::test_degrees_to_nmea PASSED
boatd/tests/test_nmea.py::test_negative_degrees_to_nmea PASSED
boatd/tests/test_nmea.py::test_gll PASSED
boatd/tests/test_plugin.py::test_find_plugins PASSED
boatd/tests/test_plugin.py::test_get_module_name PASSED
boatd/tests/test_plugin.py::test_load_plugins PASSED
boatd/tests/test_plugin.py::test_start_plugins PASSED
boatd/tests/test_utils.py::TestUtils::test_reldir PASSED
```
Reports 30 tests run, but it should be higher than that. Of note, API tests are not run.

**Comment by SamStudio8** [comment]
I'm pretty sure that for naive autodetection of tests, each test suite named `test_thing.py` needs a corresponding `thing.py`.

**Comment by kragniz** [comment]
Closed in #21. Thanks!
